dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/ThrowableProxyActivator.java

```diff
@@ -51,345 +50,343 @@
-     * Activation callback from the Declarative Services runtime where the
-     * component is ready for activation.
-     *
-     * @param bundleContext the bundleContext
-     */
-    public void activate() throws Exception {    
-        // Store a reference to the printStackTraceOverride method from BaseTraceService
-        throwableInfo = new ThrowableInfo(inst);
+	/**
+	 * Activation callback from the Declarative Services runtime where the
+	 * component is ready for activation.
+	 *
+	 * @param bundleContext the bundleContext
+	 */
+	public void activate() throws Exception {    
+	    // Store a reference to the printStackTraceOverride method from BaseTraceService
+	    throwableInfo = new ThrowableInfo(inst);
 		
-        if (throwableInfo.isInitialized()) {
-        	String runtimeVersion = getRuntimeClassVersion();
-            if (runtimeVersion != null && !runtimeVersion.equals(getCurrentVersion())) {
-                // TODO: Use a compatibility check instead
-                throw new IllegalStateException("Incompatible proxy code (version " + runtimeVersion + ")");
-            }
-
-            // Find or create the proxy jar if the runtime code isn't loaded
-            if (runtimeVersion == null) {
-    	        JarFile proxyJar = getBootProxyJarIfCurrent();
-    	        if (proxyJar == null) {
-    	            proxyJar = createBootProxyJar();
-    	        }
-    	        inst.appendToBootstrapClassLoaderSearch(proxyJar);
-            }
-            
-        	activateThrowableProxyMethodTarget();
-    
-        	ThrowableClassFileTransformer tcfTransformer = new ThrowableClassFileTransformer();
-			//for (Class<?> clazz : ) {
-			  //  if (clazz.getName().equals("java.lang.Throwable")) {
-			    	try {
-			    		inst.addTransformer(tcfTransformer, true);
-			            inst.retransformClasses(Throwable.class); //something to replace retransfornm
-			        } catch (Throwable t) {
-			            t.printStackTrace();
-			        } finally {
-			        	inst.removeTransformer(tcfTransformer);
-			        }
-			   //  }
-	//		}
-        }
-    	
+	    if (throwableInfo.isInitialized()) {
+	    	String runtimeVersion = getRuntimeClassVersion();
+	        if (runtimeVersion != null && !runtimeVersion.equals(getCurrentVersion())) {
+	            // TODO: Use a compatibility check instead
+	        	throw new IllegalStateException("Incompatible proxy code (version " + runtimeVersion + ")");
+	        }
+	
+			// Find or create the proxy jar if the runtime code isn't loaded
+			if (runtimeVersion == null) {
+			    JarFile proxyJar = getBootProxyJarIfCurrent();
+			    if (proxyJar == null) {
+			    	proxyJar = createBootProxyJar();
+			    }
+			    inst.appendToBootstrapClassLoaderSearch(proxyJar);
+			}
+	        
+	    	activateThrowableProxyMethodTarget();
+	
+	    	ThrowableClassFileTransformer tcfTransformer = new ThrowableClassFileTransformer();
+	
+	    	try {
+	    		inst.addTransformer(tcfTransformer, true);
```

Comment:
fix indent

Reply:
Fixed in [c526dc4](https://github.com/OpenLiberty/open-liberty/pull/16276/commits/c526dc432032c3e2466cddbf875e78e70c7de8da)